helm/oms-agent/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+appVersion: 0.0.1
```

Comment:
Does the oms-agent have a version? `appVersion` should reflect this version.

Reply:
yeah the version - ciprodsmthsmth

Reply:
Fixed